### PR TITLE
ACL updates

### PR DESCRIPTION
**Pull request by IAmATeaPot418**
@banker @AlonMagrafta @bmansheim - I think this will help simplify the ACL documentation for our users and make it a bit more workflow-based and explanatory.

### CODE REVIEW

File: content/rs/administering/access-control/user-roles.md [hunk 1]
```diff
@@ -5,83 +5,73 @@ weight: $weight
...
+To create a user role that only has access to the database, assign the user the management role **None**
+{{% /note %}}
 
-To control user access to Redis database commands and keys,
-you must define Redis ACLs that specify the commands that users can run and keys that the commands can apply to.
-Then, in the user role, select the databases that the users can access and Redis ACL that controls user access to those databases.
+### Database Access Controls
 
-A-A cannot assign ACLs on creation
+ACLs allow you to control what level of access each user has in Redis. With ACLs, you can specify which commands specific users can execute and which keys they can access. This allows for much better security practices: you can restrict any given user’s access to the least level of privilege needed.
```
Comment (banker) [inline content/rs/administering/access-control/user-roles.md:39]: @IAmATeaPot418 This seems to imply that you set access on a per-user basis. I think this needs some rewording.

Comment (AlonMagrafta) [reply content/rs/administering/access-control/user-roles.md:39]: How about:
**Redis** ACLs allow you to control what level of access each **role and** user have in Redis. With **Redis** ACLs, you can specify which commands specific **roles and** users can execute and which keys they can access. This allows for much better security practices: you can restrict any given user’s access to the least level of privilege needed.

File: content/rs/administering/access-control/user-roles.md [hunk 2]
```diff
@@ -5,83 +5,73 @@ weight: $weight
...
-    In Redis:
-    - These ACL subcommands are blocked: LOAD, SAVE, SETUSER, DELUSER, GENPASS, LOG
-    - These ACL subcommands are allowed: LIST, USER, GETUSER, CAT, WHOAMI, HELP
-- The MULTI, EXEC, DISCARD commands are always allowed, but ACLs are enforced on MULTI subcommands.
-- External users cannot authenticate with databases.
-- When you run multi-key commands on multi-slot keys, the return value is `failure` but the command runs on the keys that are allowed.
-
+1. The ACL subcommands LOAD, SAVE, SETUSER, DELUSER, GENPASS and LOG are blocked
+2. The MULTI, EXEC, DISCARD commands are always allowed, but ACLs are enforced within a transaction
+3. When you run multi-key commands on multi-slot keys, the return value is `failure` but the command runs on the keys that are allowed.
```
Comment (AlonMagrafta) [inline content/rs/administering/access-control/user-roles.md:46]: Let's add here that you can edit this as configuration per DB to block or allow such cases.
Using rladmin:
rladmin tune db <db:id> mkms <enabled/disabled>
Also important: inherent commands restrictions for multi-keys commands such as in OSS Cluster mode or in Active-Active will apply even when this setting (mkms) is turned ON.

Comment (IAmATeaPot418) [reply content/rs/administering/access-control/user-roles.md:46]: @AlonMagrafta - would you be able to take a swing at this. I don't understand it fully enough to do it the right level of justice.

File: content/rs/administering/access-control/user-roles.md (hunk 1 above)
Comment (IAmATeaPot418) [reply content/rs/administering/access-control/user-roles.md:39]: I like Alon's wording. Will add a commit with that if you agree Kyle.

Comment (banker) [reply content/rs/administering/access-control/user-roles.md:39]: Fine by me @IAmATeaPot418

**Comment by IAmATeaPot418** [comment]
@AlonMagrafta - LMK what you think.

File: content/rs/administering/access-control/user-roles.md (hunk 1 above)
Comment (AlonMagrafta) [reply content/rs/administering/access-control/user-roles.md]: I pushed the new wording in the last commit

File: content/rs/administering/access-control/user-roles.md (hunk 2 above)
Comment (AlonMagrafta) [reply content/rs/administering/access-control/user-roles.md]: Took it :)

**Comment by AlonMagrafta** [comment]
@IAmATeaPot418 @banker 
I pushed two more commits

**Comment by IAmATeaPot418** [comment]
@banker @bmansheim - thoughts?

**Comment by bmansheim** [comment]
I'll assume these changes will be included in the RS and RC Security overhaul.

**Pull request closed without merging**